superset-frontend/src/components/TableSelector/index.tsx

```diff
@@ -78,6 +78,10 @@ const TableLabel = styled.span`
   }
 `;
 
+const smallColor = css`
+  color: rgba(102, 102, 102, 0.7);
```

Comment:
@eschutho  do we have anything in supersetTheme with lower opacity?

Reply:
ah yeah, I think you should format it like this:
https://github.com/apache/superset/blob/pexdax/db-connection-ui/superset-frontend/src/views/CRUD/data/database/DatabaseModal/styles.ts#L46 (above that there is one that uses theme and css). 

Instead of grayscale.base try grayscale.light1? (play around with it).

Reply:
I think colors.text.help will be the next lightest color.

Reply:
And this is how you invoke it: https://github.com/apache/superset/blob/pexdax/db-connection-ui/superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx#L489